client/solo/commands/install.py

```diff
@@ -550,6 +601,9 @@ def _install_action(self, entropy_client, deps, recursive,
                 entropy_client, inst_repo,
                 run_queue, removal_queue, ask, pretend, quiet, verbose)
 
+            installed_pkg_sources = self._get_installed_packages_sources(
```

Comment:
Yet this is the only thing to get the information. See my comment (the non inline one) for rationale (i.e. "the only way I found to make this actually work".)